db/migrate/20161003033041_create_restaurants.rb

```diff
@@ -0,0 +1,11 @@
+class CreateRestaurants < ActiveRecord::Migration[5.0]
+  def change
+    create_table :restaurants do |t|
+      t.string :name
+      t.text :description
+      t.references :user, foreign_key: true
+
```

Comment:
Alright, fair enough. (I guess. You totally changed the story before working on the feature...)